python/ray/rllib/agents/ars/ars.py

```diff
@@ -94,6 +94,9 @@ def __init__(self,
                 config["observation_filter"], config["fcnet_hiddens"],
                 **policy_params)
 
+    def get_filter(self):
+        return self.policy.get_filter()
```

Comment:
It would better to use the common FilterManager sync code. To get that to work, you can copy this code adapted from the policy evaluator class here:

```
    @property
    def filters(self):
        return {"default": self.policy.get_filter()}

    def sync_filters(self, new_filters):
        for k in self.filters:
            self.filters[k].sync(new_filters[k])

    def get_filters(self, flush_after=False):
        return_filters = {}
        for k, f in self.filters.items():
            return_filters[k] = f.as_serializable()
            if flush_after:
                f.clear_buffer()
        return return_filters
```

Then, you can call on the driver process `FilterManager.synchronize({"default": self.policy.get_filter()}, self.workers)` at the end of each train().

Reply:
Oh great, I'll make this change. Thanks!